Formula/runme.rb

```diff
@@ -0,0 +1,33 @@
+class Runme < Formula
+  desc "Execute commands inside your runbooks, docs, and READMEs"
+  homepage "https://runme.dev/"
+  url "https://github.com/stateful/runme.git",
+      tag:      "v1.3.0",
+      revision: "da4fb8ff2264ad7765eda78d9ad5b125b0e6284d"
+  license "Apache-2.0"
+  head "https://github.com/stateful/runme.git", branch: "main"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = %W[
+      -s
+      -w
+      -X github.com/stateful/runme/internal/version.BuildDate=#{time.iso8601}
+      -X github.com/stateful/runme/internal/version.BuildVersion=#{version}
+      -X github.com/stateful/runme/internal/version.Commit=#{Utils.git_head}
+    ]
+
+    system "go", "build", "-o", bin, *std_go_args(ldflags: ldflags), "./main.go"
+    generate_completions_from_executable(bin/"runme", "completion", shells: [:bash, :zsh, :fish])
+  end
+
+  test do
+    system "#{bin}/runme", "--version"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
Thanks for the feedback. I added a test that actually verifies two of the most important commands for this CLI.